File: Linguagem C/menor_de_tres.c
#include <stdio.h>

int main() {


int n1, n2, n3;

printf("Primeiro valor: ");
scanf("%d", &n1);
printf("Segundo valor: ");
scanf("%d", &n2);
printf("Terceiro valor: ");
scanf("%d", &n3);

if (n1>n2 && n2>n3) {
    printf("MENOR: %d", n3);
}
else if (n3>n2 && n2>n1) {
    printf("MENOR: %d", n1);
}
else if (n1>n3 && n3>n2) {
    printf("MENOR: %d", n2);
}
else if (n2>n1 && n1>n3) {
    printf("MENOR: %d", n3);
}
else if (n2>n3 && n3>n1) {
    printf("MENOR: %d", n1);
}
else if (n3>n1 && n1>n2) {
    printf("MENOR: %d", n2);
}

return 0;
}

File: Python/menor_de_tres.py
'''Fazer um programa para ler três números inteiros. Em seguida, mostrar qual o menor dentre os três números lidos. 
Em caso de empate, mostrar apenas uma vez.'''

num1 = int(input("Primeiro valor: "))
num2 = int(input("Segundo valor: "))
num3 = int(input("Terceiro valor: "))

menor = 0

if num1 < num2 and num3:
    menor = num1

elif num2 < num1 and num3:
    menor = num2

else:
    menor = num3

print(f"Menor: {menor}")